packages/configuration-web-recommended/README.md

```diff
@@ -5,6 +5,15 @@ production websites.
 
 This package will automatically install all the missing dependencies.
 
+To use it you will have to install it via `npm`:
+
+```bash
+npm install @sonarwhal/configuration-web-recommended
+```
+
+If you want to install it globally, add the parameter `-g`.
```

Comment:
Maybe we should just link to the [npm docs](https://docs.npmjs.com/cli/install) docs?

Reply:
Or have something in the lines of:

> You can make `npm` install it as a `devDependency` using the `--save-dev` parameter, or to install it globally, you can use the `-g` parameter. For other options see the [`npm`'s documentation](https://docs.npmjs.com/cli/install).

Reply:
Do you mean something like:
`For more install options, you can see the [npm documentation](https://docs.npmjs.com/cli/install)`
?